docs/reference/connectors/gbq.md

```diff
@@ -26,14 +28,36 @@ The timeouts are in nanoseconds.
 |-----------------|------------------------------------------------------------------------------------------------------------------|
 | table_id        | The identifier of the table in the format: `projects/{project-name}/datasets/{dataset-name}/tables/{table-name}` |
 | connect_timeout | The timeout in nanoseconds for connecting to the Google API                                                      |
-| request_timeout | The timeout in nanoseconds for each request to the Google API                                                    |
+| request_timeout | The timeout in nanoseconds for each request to the Google API. A timeout hit will fail the event.                |
 
 ## Metadata
 There is no metadata needed for this connector.
 
 ## Payload structure
 
-Currently the connector is only a sink - it expects events sent to it to be objects, where the keys match the table field names, and values are the values.
+The event payload sent to the `gbq_writer` connector needs to be a [`record`](../../language/expressions.md#records) with field names being the table column names
+and the values need to correspond to the table schema values.
+
+Tremor values are mapped to Google Bigquery schema types according to the following value mapping. You need to provide the tremor value type on the right to feed a column of the left side.
+
+| Google Bigquery type (gRPC type) | Tremor Value | Format                                                    |
+|----------------------------------|--------------|-----------------------------------------------------------|
+| Numeric                          | String       | `X.Y` (no thousands separator, `.` as decimal point)      |
+| Bignumeric                       | String       | `X.Y` (no thousands separator, `.` as decimal point)      |
```

Comment:
Not including the string quotes could be missleading as ppl might copy it and expect `X.Y` to work
```suggestion
| Numeric                          | String       | `"X.Y"` (no thousands separator, `.` as decimal point)      |
| Bignumeric                       | String       | `"X.Y"` (no thousands separator, `.` as decimal point)      |
```